tests/tt_eager/python_api_testing/non_working_unit_tests/grayskull/test_sum_2.py

```diff
@@ -45,26 +45,34 @@ def run_sum_2_tests(input_shape, dtype, dlayout, in_mem_config, out_mem_config,
 
 test_sweep_args = [
     (
-        (6, 2, 216, 186),
+        (6, 2, 216, 128),
```

Comment:
Was there a reason this case we changed the shape?

Reply:
Updated